projects/jdom/Dockerfile

```diff
@@ -16,7 +16,7 @@
 
 FROM gcr.io/oss-fuzz-base/base-builder-jvm
 
-RUN curl -L https://dlcdn.apache.org//ant/binaries/apache-ant-1.10.12-bin.tar.gz -o ant.tar.gz && \
```

Comment:
Just checking: some of the changes in this PR are changes not focused on the `--instrumentation_includes` as described in the PR text, is this intentional?

Reply:
@DavidKorczynski Sorry I forgot to add that part into the description, it is updated now. Thanks for reminding :)

Reply:
Thanks!
I will convert this to a draft for now, please feel free to ping me when it is ready : )

Reply:
@Alan32Liu Thank you but actually this PR is ready to be merged, thanks for merging :)